howso/utilities/feature_attributes/pandas.py

```diff
@@ -380,6 +381,12 @@ def _as_float(value, dtype):
             if not allow_null:
                 output = {'allow_null': False}
 
+        if decimal_places:
```

Comment:
What if decimal places is 0? i.e., nearest whole number? Better to check for `if decimal_places is not None`

Reply:
good thought, will do